src/content/post/2025-07-14-kuzu-v-0.11.0.md

```diff
@@ -0,0 +1,157 @@
+title: "Kuzu 0.11.0 Release"
+description: "Release announcement for Kuzu 0.11.0"
+pubDate: "July 14 2025"
+heroImage: "/img/default.png"
+categories: ["release"]
+authors: ["team"]
+tags: ["cypher"]
+---
+
+Kuzu v0.11.0 is here! The team is excited to unveil this release, which is packed with a ton of new
+features and improvements.
+
+## Single-file databases
+
+Starting from version 0.11.0, a database in Kuzu is _just a file_ on disk. This single-file design has
+been long-adopted in other embedded database systems, such as DuckDB and SQLite, making your Kuzu
+databases much more portable and easier to share or archive. To easily distinguish your Kuzu database files 
+from other files, we recommend using the `.kuzu` or `.kz` extension.
+
+Note that this change is not backward-compatible, so you can use the [export](https://docs.kuzudb.com/migrate/)
+command to migrate your existing Kuzu database directories from older versions to v0.11.0. If you're interested,
+you can learn more about the different on-disk files created and managed by Kuzu [here](https://dev-docs.kuzudb.com/developer-guide/files/).
+
+## Performance improvements
+
+There have been numerous PRs in this release focusing on improving the performance of the vector and
+full-text search (FTS) indices. We list some of the highlights below.
+
+### Mutable vector & FTS indices
+
+Vector and full-text search (FTS) indices in Kuzu are now _mutable_, meaning that the index is updated the moment new data
+is added to the graph. Once an index is created on a table, any further insertion and deletion operations
+in that table will automatically trigger an update in the associated index.
+
+If your workflows involve manually dropping and recreating indices on tables that are updated
+with new data, now, you can just focus on inserting the data, and Kuzu will take care of the rest.
+
+### Filtered vector search with arbitrary Cypher queries
+
+Our initial version of vector search provided the ability to perform filtered search with simple predicates
+on a single table. In v0.11.0, we generalize the predicate to be an arbitrary Cypher query. Consider the following
+graph with a `(:Book)-[:PublishedBy]->(:Publisher)` relationship, and a vector index on the `Book` table.
+
+```sql
+CREATE NODE TABLE Book(id SERIAL PRIMARY KEY, title STRING, title_embedding FLOAT[384]);
+CREATE NODE TABLE Publisher(name STRING PRIMARY KEY);
+CREATE REL TABLE PublishedBy(FROM Book TO Publisher);
+CALL CREATE_VECTOR_INDEX(
+    'Book', -- table name
+    'book_title_index', -- index name
+    'title_embedding' -- property name on which the vector index is created
+);
+```
+
+To retrieve relevant books from a certain publisher, e.g. `Pearson`, you simply define the predicate
+using a Cypher query. The example below shows a projected graph, defined via `PROJECT_GRAPH_CYPHER`,
+and reference the name of the projected graph in the `QUERY_VECTOR_INDEX` call.
+
+```sql
+CALL PROJECT_GRAPH_CYPHER(
```

Comment:
@andyfengHKU, this command is correct, right? Your earlier version had it as `CREATE PROJECT_GRAPH_CYPHER` but the docs say we should use `CALL` for this.